### DISCUSSION TITLE
求助，我的KSU刷模块时候需要加载很长时间

### DISCUSSION THREAD

**Discussion by Do-Do666** [tiann/KernelSU#1863] [Q&A]
不想把根目录模块全部删掉重新刷，有什么办法可以解决吗

刷模块删模块还有开关模块时候加载很长时间，现在是最新版的KSU内核

**Comment by Separatee** [comment]
这是打包镜像 IO密集的操作 实属正常 装的插件越多 每次打包时间越长。
